src/style.css

```
@@ -1,3 +1,63 @@
 body {
   margin: 0;
+
+  --color-blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  justify-content: space-between;
+}
+
+.logo__image {
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style: none;
+  align-items: center;
+}
+
+.nav__link {
+  font-family: Roboto, sans-serif;
+  color: #000;
+  padding: 23px 0;
+  margin-left: 20px;
+
+  text-decoration: none;
+  cursor: pointer;
+  display: block;
+}
+
+a:hover {
```

Use only class selector
```suggestion
.nav__link:hover {
```